src/bench_schnorrsig.c

```diff
@@ -60,7 +61,7 @@ int main(void) {
 
```

Comment:
```suggestion
    CHECK(MSGLEN >= 2);
```

Reply:
There was no intention to allow users to change MSGLEN and this wouldn't work since the benchmark only used `schnorrsig_sign`, not `schnorrsig_sign_custom`. It may be useful though, so I added a commit to allow that.